### DISCUSSION TITLE
Lost weather data

### DISCUSSION THREAD

**Discussion by anderep** [jasonacox/Powerwall-Dashboard#358] [Q&A] [answered]
My weather data dropped out about -10days ago. Rebooted raspberry pi, still nothing. Have I used up my openweather access?

**Comment by anderep** [comment]
Openweather key says “active”…

**Comment by anderep** [comment, marked as answer]
Either reboot or logging into my openweather account fixed it.

**Reply by jasonacox** [reply]
Interesting.  Did anything show up running `./verify.sh` or `docker logs weather411`?
